docs/ref/models/fields.txt

```diff
@@ -235,8 +235,9 @@ modifications:
 * A ``.label`` property is added on values, to return the human-readable name.
 * A number of custom properties are added to the enumeration classes --
   ``.choices``, ``.labels``, ``.values``, and ``.names`` -- to make it easier
-  to access lists of those separate parts of the enumeration. Use ``.choices``
-  as a suitable value to pass to :attr:`~Field.choices` in a field definition.
+  to access lists of those separate parts of the enumeration.
+  To use a ```Choices`` class on :attr:`~Field.choices`, you can either use it
+  directly, or you can manually expand the value list with ``.choices``.
```

Comment:
I think there no more need to mention `.choices` here. I'd remove _"Use ``.choices`` as a suitable value to pass to :attr:`~Field.choices` in a field definition."_

```suggestion
  to access lists of those separate parts of the enumeration.
  
```